tests/test.py

```diff
@@ -89,17 +89,17 @@ def test_stopping_vm(self):
         self.assertEqual(0, process.returncode)
         self.assertEqual('Stopped', output.rstrip())
 
-    def test_swapspace(self):
-        ''' Test if swap space is available on restart '''
-        start_args = (self.driver_name, "file://"  + self.iso_file)
-        cmd = self.bin_dir + "minishift start --vm-driver %s --iso-url %s" % start_args
-        self.execute_test({ 'cmd': cmd })
-
-        # Check swap space
-        cmd = self.bin_dir + "minishift ssh \"echo `free | tail -n 1 | awk '{print $2}'`\""
-        self.log.info("Executing command : %s" % cmd)
-        output = self.execute_test({ 'cmd': cmd })
-        self.assertNotEqual(0, output)
+#    def test_swapspace(self):
```

Comment:
@gbraad , we are commenting it out or you forgot to uncomment it?

Reply:
test.py does not run...

and this PR is under test